AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Itineraries/AddRequestsCommandHandlerAsync.cs

```diff
@@ -4,46 +4,41 @@
 using System.Linq;
 using System;
 using System.Collections.Generic;
+using AllReady.Areas.Admin.Features.Requests;
 using Microsoft.EntityFrameworkCore;
 
 namespace AllReady.Areas.Admin.Features.Itineraries
 {
     public class AddRequestsCommandHandlerAsync : IAsyncRequestHandler<AddRequestsCommand, bool>
     {
-        private readonly AllReadyContext _context;
+        private readonly IAllReadyDataAccess _data;
         private readonly IMediator _mediator;
 
-        public AddRequestsCommandHandlerAsync(AllReadyContext context, IMediator mediator)
+        public AddRequestsCommandHandlerAsync( IAllReadyDataAccess data, IMediator mediator )
```

Comment:
What is the driver to avoid using the context in the handler? I've made another comment about moving away from the `IAllReadyDataAccess` class as well, feel free to explore this in the project and reach out as required. We're happy to help :)